spec/features/search_spec.rb

```diff
@@ -85,6 +85,6 @@ def submit_search
   end
 
   def order_row_match(order)
-    %r{#{order.id} #{order.customer.name} }
+    /#{order.id}\s+#{order.customer.name}\s+/
```

Comment:
Was this broken with the Capybara change too?

Reply:
Yes, the new version had put tabulation chars instead of just spaces here.